hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -247,6 +258,16 @@ public HoodieRecord generateUpdateRecord(HoodieKey key, String commitTime) throw
     return updates;
   }
 
+  /**
+   * Generates 1 new update, given the hoodie key, riderName and driverName
+   */
```

Comment:
can we call `generateUpdates(commitTime, 1)` ? Please try to reuse the existing methods as much as possible